scout/constants/acmg.py

```diff
@@ -147,36 +151,40 @@
                     (
                         "PP1",
                         {
-                            "short": "Cosegregation (WQ)",
```

Comment:
"WQ" is likely for "the evidence is weak". Using Biesecker 2023 for strenght modulation is recommended instead.